packages/mongo/collection_tests.js

```diff
@@ -89,7 +89,13 @@ Tinytest.add('collection - call native find with sort function',
     if (Meteor.isServer) {
       test.throws(
         function () {
-          nativeCollection.find({}, {sort: function(){}}).map(function (doc) { return doc.a; })
+          nativeCollection
+            .find({}, {
+              sort: function () {},
```

Comment:
Hi @benjamn ,
Thank you for merge.
I wish to use ES7 syntax, but most of file seems to be ES5. Am I wrong?